src/dashboard-refactor/header/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { PureComponent } from 'react'
+import Margin from 'src/dashboard-refactor/components/Margin'
+import styled from 'styled-components'
+
+import colors from '../../colors'
+import { fonts } from '../../styleConstants'
+
+interface SearchBarProps {
+    handleSearchBoxClick(): void
+    handleInputChange(evt): void
```

Comment:
No type information about the event here. Generally, I never pass the DOM event itself up, rather something like `onSearchQueryChange(event: { newQuery: string }): void`

Reply:
This was an error. This component isn't actually complete, I didn't realise that I merged it into this branch sorry.